static/usage/v7/select/start-end-slots/demo.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Select</title>
+    <link rel="stylesheet" href="../../../common.css" />
+    <script src="../../../common.js"></script>
+    <script type="module" src="https://cdn.jsdelivr.net/npm/@ionic/core@7/dist/ionic/ionic.esm.js"></script>
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@ionic/core@7/css/ionic.bundle.css" />
+
+    <style>
+      ion-list {
+        min-width: 400px;
+      }
+    </style>
```

Comment:
This is needed to handle a bug introduced in `feature-7.6` where the select's width collapses to 0 when in an item wrapped in a flex container. We'll address this bug separately.

Reply:
Maybe there should be a comment to make sure it doesn't get removed until there's a fix.